common/src/main/java/net/william278/husksync/config/Settings.java

```diff
@@ -121,6 +122,7 @@ public enum ConfigOption {
         CLUSTER_ID("cluster_id", OptionType.STRING, ""),
         DEBUG_LOGGING("debug_logging", OptionType.BOOLEAN, false),
 
+        DATABASE_TYPE("database.type", OptionType.STRING, DatabaseType.MYSQL.name()),
         DATABASE_HOST("database.credentials.host", OptionType.STRING, "localhost"),
```

Comment:
Let's rename the MYSQL credential parameters to `DATABASE_MYSQL_<param>` -- and don't forget to update the key here when you update this in config.yml, too.